.github/workflows/helm-chart-ci.yaml

```diff
@@ -32,37 +32,76 @@ jobs:
       - name: helm lint
         run: |
           cd helm-chart/
-          helm lint
+          helm lint --strict
```

Comment:
Praise: This [`--strict` flag](https://helm.sh/docs/helm/helm_lint/) ("fail on lint warnings") is a good idea!